NUsight2.sublime-project

```diff
@@ -0,0 +1,16 @@
+{
+	"folders":
+	[
+        {
+            "path": "."
```

Comment:
Surely this isn't still needed, is it needed? If so, can we keep the indentation consistent (2 spaces)?

Reply:
path: "." is still needed for it to show up properly